static/css/style.css

```diff
@@ -23,14 +23,17 @@ body{
   padding-bottom: 3vmin;
 }
 
+.line1Box{
```

Comment:
add a comma then line2box if we want to apply these styles to both elements, or conversely, use only one class in the HTML